db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` for `genres` field in `Movie` model is not correct. `related_name` is used to create a backward relation from `Genre` to `Movie`. So it should be something like `movies` instead of `genres`. This is a code style issue.